gcp-observability/src/main/java/io/grpc/gcp/observability/ObservabilityConfig.java

```diff
@@ -54,27 +55,49 @@ public interface ObservabilityConfig {
   /**
    * POJO for representing a filter used in configuration.
    */
+  @ThreadSafe
   class LogFilter {
-    /** Pattern indicating which service/method to log. */
-    public final String pattern;
+    /** Pattern for service/method filter. */
+    public final List<String> pattern;
```

Comment:
What is this used for?

Reply:
This is just used to store patterns as we see in config. Since we already parse it, I will remove.